src/SmartComponents/RunTaskModal/hooks/useBulkSystemSelect.js

```diff
@@ -17,7 +17,7 @@ export const useSystemBulkSelect = (
 
       case 'page': {
         options.items.forEach((item) => {
-          if (!newSelectedIds.includes(item.id)) {
+          if (!newSelectedIds.includes(item.id) && !item.requirements.length) {
```

Comment:
A nitpick: curious why we need `!item.requirements.length` here ^. I believe we can make this more readable by having a new constant for this boolean and putting it inside the if. 

For example: `const someRrequirementsShouldBeEmpty = !item.requirements.length`

Reply:
Fixed!  Better readability is always a good thing :)